extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/theta/sql/ThetaSketchSetBaseOperatorConversion.java

```diff
@@ -63,6 +63,8 @@ public DruidExpression toDruidExpression(
       RexNode rexNode
   )
   {
+    plannerContext.setPlanningError("%s can only be used on aggregates. " +
+        "It cannot be used directly on a column or or on a scalar expression.", getFunctionName());
```

Comment:
typo: double `or`

Since the error message is specific to theta sketch, maybe we can mention the exact type of arguments this function expects, rather than mentioning what it cannot accept.

Reply:
the type check is already done. This tells the user that they are trying to use the function as an expression while its only available as a post aggregator.

Reply:
Ah, okay. Thanks for the clarification.